### ISSUE TITLE
Always return  " Invalid progress type: {"responseId":"xxx"} "

### ISSUE THREAD

**Issue by Wenbobobo** [microsoft/vscode-copilot-release#585]
GitHub Copilot Chat

- VS Code Version: 1.85.0-insider(user setup)
- OS Version:Windows 100.19044
- GitHub Copilot: v1.137.0
- GitHub Copilot Chat: v0.10.1
- Logs:(Copilot Chat)
2023-11-20T08:46:56.521Z [INFO] [auth] Logged in as Wenbobobo
2023-11-20T08:46:57.215Z [INFO] [chat] copilot token chat_enabled: true
2023-11-20T08:46:57.218Z [INFO] [auth] Got Copilot token for Wenbobobo
2023-11-20T08:46:57.234Z [INFO] [auth] Logged in as Wenbobobo
2023-11-20T08:46:57.236Z [INFO] [chat] copilot token chat_enabled: true
2023-11-20T08:46:57.236Z [INFO] [auth] Got Copilot token for Wenbobobo
2023-11-20T08:47:01.012Z [ERROR] [extension] (Error) Invalid progress type: {"responseId":"535a0277-3589-493c-9682-351f33782719"}
2023-11-20T08:47:18.213Z [ERROR] [extension] (Error) Invalid progress type: {"responseId":"6b1cecc8-3d3e-4f22-a922-ad70b409abee"}

Steps to Reproduce:

The copilot is okay but the chat will  return " Invalid progress type: {"responseId":"xxx"} " whatever i input
![image](https://github.com/microsoft/vscode-copilot-release/assets/78262508/97352964-fe25-437b-81f5-100709e1a3ed)

**Comment by DuncanLHS** [comment]
I've also had this error today.

Initially, copilot was unavailable altogether and had to go enable "GitHub Copilot Chat for IDE" in my orgs account settings, this appears to be a new setting that should've been enabled by default considering I was using it just a few days ago.

Then I got the error above, it appears to be related to using the release version of copilot in vscode insiders, switching to pre-release version of the copilot and copilot chat extensions resolved the issue. Both versions of the extensions appear to be working in standard vscode.

**Comment by Wenbobobo** [comment]
> go enable "GitHub Copilot Chat for IDE" in my orgs account settings

but i dont have an org yet 😥

**Comment by lsmith-apriva** [comment]
>  it appears to be related to using the release version of copilot in vscode insiders, switching to pre-release version of the copilot and copilot chat extensions resolved the issue. Both versions of the extensions appear to be working in standard vscode.

Piling onto DuncanLHS's reply: I was previously operational with CoPilot.  When I started work today, it was no longer working.   I was using the pre-release GitHub Copilot extentions in VS Code Preview edition.  
I noticed that the CoPilot extentions had switched themselves to be the normal channel ones.
I switched them back to pre-release and rebooted VS Code and now they work again. No changes needed to my GitHub Org.
